<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>569</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="deviceSelectLayout">
       <item>
        <spacer name="deviceSelectSpacerLeft">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="deviceSelectLabel">
         <property name="text">
          <string>ST-LINK S/N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="deviceSelectBox">
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deviceSelectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="deviceSelectSpacerRight">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="protocolSelect">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuNumber_Base">
     <property name="title">
      <string>Number Base</string>
     </property>
     <addaction name="actionBinary_2"/>
     <addaction name="actionOctal_8"/>
     <addaction name="actionDecimal_10"/>
     <addaction name="actionHexadecimal_16"/>
    </widget>
    <addaction name="menuNumber_Base"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBinary_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Binary (2)</string>
   </property>
  </action>
  <action name="actionOctal_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Octal (8)</string>
   </property>
  </action>
  <action name="actionDecimal_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Decimal (10)</string>
   </property>
  </action>
  <action name="actionHexadecimal_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hexadecimal (16)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
